catanatron_core/catanatron/state_functions.py

```diff
@@ -134,6 +134,17 @@ def get_player_buildings(state, color_param, building_type_param):
     return state.buildings_by_color[color_param][building_type_param]
 
 
+def get_player_freqdeck(state, color):
+    key = player_key(state, color)
+    return [
+        state.player_state[f"{key}_WOOD_IN_HAND"],
+        state.player_state[f"{key}_BRICK_IN_HAND"],
+        state.player_state[f"{key}_SHEEP_IN_HAND"],
+        state.player_state[f"{key}_WHEAT_IN_HAND"],
+        state.player_state[f"{key}_ORE_IN_HAND"],
+    ]
```

Comment:
can you add a description/docstring for this function?

Reply:
Yep.